scripts/vaccination_alerts/vaccination-alert-template-source.html

```diff
@@ -0,0 +1,247 @@
+<html>
```

Comment:
Hrm. Can you add a note to the readme to explain what's up with this -source.html file vs the main .html file?  Seems this isn't directly used anywhere?